extensions/amp-fit-text/0.1/amp-fit-text.js

```diff
@@ -224,6 +226,7 @@ export function updateOverflow_(content, measurer, maxHeight, fontSize) {
  *
  * @param {!Document} document
  * @param {!Element} element
+ * @return {{content: !Element, contentWrapper: !Element, measurer: !Element}}
```

Comment:
I'm fine with this as-is for now, but once we add the SSR'd path it'll be a good pattern to name this type so it can be referenced in a couple places at once